go/libraries/doltcore/sqle/procedures_table.go

```diff
@@ -94,6 +101,88 @@ func DoltProceduresGetOrCreateTable(ctx *sql.Context, db Database) (*WritableDol
 	return tbl.(*WritableDoltTable), nil
 }
 
+// TODO: godocs
+// TODO: what's the best way to test this? backwards compat test suite?
+func migrateDoltProceduresSchema(ctx *sql.Context, db Database, oldTable *WritableDoltTable) (newTable *WritableDoltTable, rerr error) {
```

Comment:
I still need to come up with a way to test the migration code path. I saw some tests for dolt_schema migration yesterday, so I'll dig into those more next.